core/src/main/java/nu/yona/server/subscriptions/service/BuddyDisconnectMessageDTO.java

```diff
@@ -106,7 +106,15 @@ private MessageActionDTO handleAction_Process(UserDTO actingUser, BuddyDisconnec
 		{
 			buddyService.removeBuddyAfterBuddyRemovedConnection(actingUser.getID(), messageEntity.getUserID());
 
+			updateMessageStatusAsProcessed(messageEntity);
```

Comment:
This fixes a bug: the message status didn't flip to "processed" after processing it.